packages/transactions/src/utils.ts

```diff
@@ -10,6 +10,9 @@ import lodashCloneDeep from 'lodash.clonedeep';
 import { with0x } from '@stacks/common';
 import { bytesToHex } from '@noble/hashes/utils';
 
+// Export verify as utility method for signature verification
+export { verify } from '@noble/secp256k1';
```

Comment:
```suggestion
export { verify as verifySignature } from '@noble/secp256k1';
```
can we rename this to make it more clear that a signature is being verified?